docs/extension.rst

```diff
@@ -24,12 +25,15 @@ Using the extension
 
 The extension can be found on the `Visual Studio Marketplace <https://marketplace.visualstudio.com/items?itemName=solang.solang>`_.
 
-On first start, the extension will ask to download the Solang binary. Once this is done, it should just automatically work.
+When started for the first time, the extension will download the Solang binary. Once this is done, it should just automatically work.
 However, you should set the blockchain target in the extension settings. The Solidity language differs in subtle ways depending on which target you are
```

Comment:
To my understanding, the Syntax solang supports is a superset of Solidity? If that is true I think we should point that out here. Also we maybe we should say what version we support somewhere.

Reply:
Good point, I've added another section for this